tslint-server/src/server.ts

```diff
@@ -109,12 +109,15 @@ folder using \'npm install tslint\' or \'npm install -g tslint\' and then press
 // Options passed to tslint
 let options: tslint.ILinterOptions = {
 	formatter: "json",
-	configuration: {},
+	fix: false,
```

Comment:
Not sure I understand. I pass in a configuration that is retrieved from `findConfiguration` that is then cached. 
```typescript
linterConfiguration.findConfiguration(configFileName, filePath).results
```